.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: "20.0.0"
+          node-version: "22.6.0"
```

Comment:
node `22` is not fully supported by many node modules yet. Especially next js

Reply:
Unfortunately, it's needed for some ICPs peer dependencies to work